### ISSUE TITLE
Tool tip showing but is invisible 

### ISSUE THREAD

**Issue by Shane-01** [Thrilleratplay/jquery-validation-bootstrap-tooltip#40]
I'm trying to use your jquery validation to display my error messages in the tool tip, the problem i have is that the tool tip is invisible please check the image I attached.
Any help would be greatly appreciated
This is the code I have in my view.

    ViewData["Title"] = "About";
}

<head>
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
</head>
<h2>@ViewData["Title"]</h2>
<h3>@ViewData["Message"]</h3>

<p>Use this area to provide additional information.</p>
<button type="button" class="btn btn-secondary" data-toggle="tooltip" data-placement="right" title="Tooltip on right">
    Tooltip on right
</button>

<form id="Form">
    <div class="form-group">
        <label class="control-label">Test</label>
        <input id="FromAddress" name="FromAddress" />

    </div>
    <input type="submit" value="Test"/>

</form>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.bundle.min.js" integrity="sha384-3ziFidFTgxJXHMDttyPJKDuTlmxJlwbSkojudK/CkRqKDOmeSbN6KLrGdrBQnT2n" crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/jquery-validation@1.17.0/dist/jquery.validate.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/jquery-validation@1.17.0/dist/additional-methods.min.js"></script>
<script src="~/lib/jquery-validate-bootstrap-tooltip/jquery-validate.bootstrap-tooltip.js"></script>
<script>
    //<img src="~/images/Artboard 2.png" height="20" width="20" />

    $(function () {
        $('[data-toggle="tooltip"]').tooltip();
    });

    var validator = $("#Form").validate({
        rules: {
            FromAddress: { required: true }
        },
        messages: {
            FromAddress: "Please Fill In this feild"
        },
        tooltip_options: {
            FromAddress: { trigger: 'hover focus', placement: 'right' }

        },
    });
</script>

![problem](https://user-images.githubusercontent.com/31413661/41594382-85281134-73c3-11e8-93cc-8aee673ba698.PNG)

**Comment by Thrilleratplay** [comment]
@Shane-01   Can you create a [JSfiddle](http://jsfiddle.net/) reproducing this?

**Comment by Shane-01** [comment]
@Thrilleratplay  I Fixed the issue , I had different versions of bootstrap in my solution, so i made sure all references of bootstrap are 3.3.7 and it seems to be working fine now.

**Comment by Thrilleratplay** [comment]
Bootstrap made a few changes around v3.3.x that caused issues with this plugin.  Glad you fixed the issue.
